core/src/main/java/com/linecorp/armeria/internal/common/AbstractKeepAliveHandler.java

```diff
@@ -46,6 +48,8 @@ public abstract class AbstractKeepAliveHandler implements KeepAliveHandler {
 
     private static final Logger logger = LoggerFactory.getLogger(AbstractKeepAliveHandler.class);
 
+    private static final AtomicInteger numActiveConnections = new AtomicInteger();
```

Comment:
If a `client` and `server` are running in the same jvm, I feel like this could be double-counted